x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/details_step_form.tsx

```diff
@@ -59,6 +61,23 @@ export const DetailsStepForm: FC<CreateAnalyticsStepProps> = ({
     destinationIndexNameValid === false ||
     (destinationIndexPatternTitleExists === true && createIndexPattern === true);
 
+  const debouncedIndexCheck = debounce(async () => {
+    const { indexExists } = await ml.checkIndexExists({ index: destinationIndex });
+    setFormState({ destinationIndexNameExists: indexExists ?? false });
```

Comment:
It looks like the ?? isn't needed here. The response will with be a boolean or an error will be thrown.
The error should be caught and a toast displayed.

Reply:
Agreed. Removed the ?? operator and caught error with added toast display e95d7ddcdae1485a589cc3b90e0bbd163ad22621